kubermatic/resource_project.go

```diff
@@ -119,37 +121,18 @@ func resourceProjectRead(d *schema.ResourceData, m interface{}) error {
 		return fmt.Errorf("unable to get project '%s': %v", d.Id(), err)
 	}
 
-	err = d.Set("name", r.Payload.Name)
-	if err != nil {
+	if err := d.Set("labels", r.Payload.Labels); err != nil {
 		return err
 	}
-
-	err = d.Set("labels", r.Payload.Labels)
-	if err != nil {
-		return err
-	}
-
-	err = d.Set("status", r.Payload.Status)
-	if err != nil {
-		return err
-	}
-
-	err = d.Set("creation_timestamp", r.Payload.CreationTimestamp.String())
-	if err != nil {
-		return err
-	}
-
-	err = d.Set("deletion_timestamp", r.Payload.DeletionTimestamp.String())
-	if err != nil {
-		return err
-	}
-
+	d.Set("name", r.Payload.Name)
```

Comment:
any reason why you don't check errors here?

Reply:
https://www.terraform.io/docs/extend/best-practices/detecting-drift.html#error-checking-aggregate-types

Reply:
right, I missed that

> The preceding examples featured primitive types which don't require error checking